### DISCUSSION TITLE
Log panel: With shared crosshair log panel should scroll into logs with hovered-over time

### DISCUSSION THREAD

**Discussion by pokemane** [grafana/grafana#58411] [Dashboards, Panels & Visualizations]
**What would you like to be added**:

I'd like for the Logs panel to use the "shared crosshair" dashboard option, to sync the selected/focused line with the graph crosshairs. It'd also want to autoscroll the log panel to have that line in the center of the panel vertically.

**Why is this needed**:

It'd really help with the usability of the log panel, to be able to more easily correlate log events with the related graphs elsewhere on the dashboard.

I know that there might be some weirdness when you're hovering over a line in the log, where it might not be exactly clear where to put the crosshair or tooltip on the graphs. I'd be okay with it being "one-way" so the log panel can't pop the crosshairs up on the graphs, but the graphs *can* highlight or focus a line in the log panel. **EDIT** This is probably applicable to the Explore log view too.

**Comment by aocenas** [comment]
@davkal this seems like an interesting feature and maybe not even that complicated. Do you know whether something like this was already discussed or requested somewhere else?

**Comment by davkal** [comment]
@aocenas yup, this absolutely sounds helpful. It's not discussed anywhere else AFAIK. I've been wanting to do something similar across Explore splits, but it's not written down anywhere.

**Comment by rplevka** [comment]
I was just about to open the same RFE (including autoscroll). I was surprised this wasn't implemented yet. It would be really helpful!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). I believe this feature has been implemented. If you're not finding a solution for your use case, feel free to make an [issue](https://github.com/grafana/grafana/issues) to request the feature.
